1907-01-21.xml

```diff
@@ -1494,13 +1497,979 @@
+                        the current quarter by another £700,000; the total decrease will still fall
+                        far short of the estimate. Excise, estimated to decrease by £30,000, shows
+                        for the nine months an increase of £107,000. The Death-duties were, assumed
+                        t> grow by £230,000, but the actual growth during threo-querters of the
+                        financial year has been £1,495,000. The increase in the Income-tax Returns
+                        was patat £150,000 ; it is already £167,000 And in "miscellaneous items" an
+                        estimated deficit oh the year of ,£13,000 haa been changed to an increase on
+                        the nine months of £336,800. One or two items, such as stamps, show more
+                        than their eitimated decrease, and-it looks as if the Post Office and
+                        Telegraph receipts might fall short of their estimated increase. But these
+                        are minor figures com paled with the great totals of incroase. Mr Asquith
+                        can scarcely avoid having in hand at the end.of the financial year a balance
+                        over his estimates of something between £2,000,000 and £3,000,000, and if he
+                        succeeds in effecting certain savings in expenditure, as seems probable, he
+                        may have a surplus of from £6,000,000 to £6,000.000 At present, of course,
+                        the trade prosperity is not lully rejected in the Revenue Returns, except to
+                        some small degree in Customs. But a prosperous commerce, high wages, and
+                        continuous employment are bound very-soon to give a new elasticity to the
+                        revenue and next year we may look to see its effects. It is true that the
+                        enormous receipts from .Death-loties last year were rather in the bature of
+                        a windfall, arid that millionaires cannot be expected to die with such
+                        regularity and profusion in every financial year. The objection is fair, but
+                        it must not be pressed too for. It seems to us that evegy Chancellor of the
+                        Exchequer is justified in Hooking for much larger receipts from the
+                        Death-duties during he coming decade than have accrued during he past The
+                        great fortunes have been most f them made within the last thirty years, and
+                        heir possessors are only just beginning to dirt Ve should not be surprised
+                        to see the Death-utiea double their yield during the next wenty years.</p>
+                </div>
+            </div>
+            </div>
+            <div type="page" n="3">
+            </div>
```

Comment:
this `</div>`, which ends the page division, should come just before the start of page 4 (i.e. after all of page 3's contents).